src/native-http-backend.ts

```diff
@@ -167,6 +148,45 @@ export class NativeHttpBackend implements HttpBackend {
         });
     }
 
+    private buildRequestOptions(req: HttpRequest<any>, requestMethod, headers) {
+        let serializerType = this.detectDataSerializerType(req);
+        const requestOptions: any = {
+            method: requestMethod,
+            headers: { ...headers },
+            serializer: serializerType,
+            body: this.convertBody(req.body, serializerType),
```

Comment:
There is no `body` in the available params list of `sendRequest` https://github.com/silkimen/cordova-plugin-advanced-http#sendrequest

Are you sure it is working?

Reply:
my bad - the option is supposed to be `data`.
I will fix this and do some more testing, especially for post request.

Reply:
it should be either data either params depending on request type.

Reply:
and it should be tested with at least `get` and `post` since `get` uses `params` and `post` uses `data`.

Reply:
The fact of switching between `params` and `data` should be reflected in unit tests.

Reply:
The `params` and `data` option should now be used for the correct request methods.

Due too using the `params` option, `request.url` is used instead of `request.urlWithParams`.